src/core/stdc/complex.d

```diff
@@ -1,6 +1,11 @@
 /**
  * D header file for C99.
  *
+ * This contains bindings to selected types and functions from the standard C
+ * header $(LINK2 http://pubs.opengroup.org/onlinepubs/009695399/basedefs/complex.h.html, <complex.h>). Note
+ * that this is not automatically generated, and may omit some types/functions
+ * from the original C header.
```

Comment:
Looks good! This seems to be repeated (modulo link), so could you please put it in a macro that we can update wholesale?

````
C_HEADER_DESCRIPTION = This module contains bindings to selected types and functions from the standard C header $(HTTP pubs.opengroup.org/onlinepubs/009695399/basedefs/$1.html, $(LT)$1$(GT)). Note that this is not automatically generated, and may omit some types/functions from the original C header.
```

Plant this bad boy in html.ddoc and expand it in all these modules.